### PR TITLE
docs: Adopt the script docs to create certs

### PR DESCRIPTION
**Pull request by ZPascal**
@chombium Do you think that's enough documentation or should I prepare a detailed step by step manual?

### CODE REVIEW

File: scripts/ssl/EXAMPLE-CERT-CREATION.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 Please execute the following script `bash create-tls-certs.sh` to create all necessary certificates for the complete setup of all related components.
 
+## Adoption
```
Comment (chombium) [inline scripts/ssl/EXAMPLE-CERT-CREATION.md:7]: Adoption means start to use something. I guess you need adaptation = adjust something, somehow, to your needs in this case

Comment (ZPascal) [reply scripts/ssl/EXAMPLE-CERT-CREATION.md:7]: Yes, it's a typo :) I'll fix it.

File: scripts/ssl/EXAMPLE-CERT-CREATION.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 Please execute the following script `bash create-tls-certs.sh` to create all necessary certificates for the complete setup of all related components.
 
-## Adoption
+## Adaptation
 
 Please adjust the configuration files inside the [config](./config) folder and adopt the corresponding values for the req_distinguished_names and subjectAltNames based on your organisation and configuration.
```
Comment (chombium) [inline scripts/ssl/EXAMPLE-CERT-CREATION.md:9]: there is another `adopt` here `...folder and adopt the corresponding values...` :-/

Comment (ZPascal) [reply scripts/ssl/EXAMPLE-CERT-CREATION.md]: Thank you. Done

**Review by chombium — APPROVED** [review]

**Pull request merged**